src/cdk/drag-drop/directives/drag.spec.ts

```diff
@@ -1842,14 +1842,22 @@ describe('CdkDrag', () => {
           body: document.body,
           fullscreenElement: document.createElement('div'),
           ELEMENT_NODE: Node.ELEMENT_NODE,
-          querySelectorAll: function() {
-            return document.querySelectorAll.apply(document, arguments);
+          querySelectorAll: function(...args: [string]) {
```

Comment:
Shouldn't it be `string[]`?

Reply:
`string[]` is a string array of any length, `[string]` is an array of exactly one string

Reply:
If we know it'll be one string, couldn't we just declare it like this?

```
querySelectorAll: (selector: string) {
  return document.querySelectorAll.call(document, selector);
}
```

Reply:
My thinking was that for these ones where we're just wrapping a function and passing things through, the author was trying to be defensive against future browser API changes by just grabbing all of the arguments and shoving them through. This style preserves that

Reply:
I'm pretty sure that I'm the author and I think I did it because it was shorter 😄

Reply:
I can only speak for the ZoneJS thing where I did it that way. I wanted to be on the safe side if a different ZoneJS version is used / when the signature changes. The typings are manually copied.

Reply:
Yeah `querySelectorAll` probably isn't going to change much, but some do, `addEventListener` has definitely changed over the years